src/app/login/login.component.ts

```diff
@@ -11,21 +12,25 @@ export class LoginComponent implements OnInit {
 
   email;
   password;
+  errorMessage;
 
-  constructor(private  UserService: UserService) { }
+  constructor(private  UserService: UserService,private Router:Router) { }
 
   ngOnInit() {
   }
 
   login(){
-    console.log("email:"+this.email+"-pass:"+this.password);
-    // this.UserService.login(this.email,this.password).then(function(response){
-    //   console.log(response);
-    // });
-    this.UserService.login(this.email,this.password).subscribe( data => {
-      console.log(data);
+    this.UserService.login(this.email, this.password).subscribe( data => {
+      console.log(data.data);
+      if(data.data) {
+      localStorage.setItem('isLoggedin', 'true');
+      localStorage.setItem('user', JSON.stringify(data.data));
```

Comment:
@BilalAlam173 arent we using JSON Webtokens for authentication? We need that for authentication